docs/topics/operation-arguments.md

```diff
@@ -65,7 +66,7 @@ For this, configure the generator for a `key: Int` parameter:
        fun get(key: K): List<V> = map[key] ?: emptyList()
    }
    
-   @Param(name = "key", gen = IntGen::class, conf = "1:2")
+   @Param(name = "key", gen = IntGen::class, conf = "1:1")
```

Comment:
I also get an expected error when I use `"1:3"` for `conf`. But the mentioned output for `modelCheckingTest` in the guide looks like `"1:1"` should be used.

Is there a reason why the original `"1:2"` doesn't work? I have an Mac M1, if that matters.

Reply:
Can you provide more details about the error, please? |

The configuration is correct, "1:1" will always generate the same key.

Reply:
>Can you provide more details about the error, please?

I am not sure whether you have requested the output for case `conf = "1:3"`. Therefore, here are the outputs for all three cases.

`stressTest` with `conf = "1:3"` for operation `get` and `add`:
```
= Invalid execution results =
Parallel part:
| add(3, 6): void | add(3, -8): void |
Post part:
[get(3): [-8]]
org.jetbrains.kotlinx.lincheck.LincheckAssertionError: 
= Invalid execution results =
Parallel part:
| add(3, 6): void | add(3, -8): void |
Post part:
[get(3): [-8]]
``` 

`stressTest` with `conf = "1:2"` for operation `get` and `add`:
_No error, the test is successful._


`stressTest` with `conf = "1:1"` for operation `get` and `add`:
```
= Invalid execution results =
Parallel part:
| add(1, 6): void | add(1, -8): void |
Post part:
[get(1): [-8]]
org.jetbrains.kotlinx.lincheck.LincheckAssertionError: 
= Invalid execution results =
Parallel part:
| add(1, 6): void | add(1, -8): void |
Post part:
[get(1): [-8]]
```

Reply:
@ndkoval Do you need anything else?